src/modules/Elsa.Workflows.Management/Providers/AssemblyPathActivityProvider.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Elsa.Workflows.Core.Contracts;
+using Elsa.Workflows.Core.Models;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
+namespace Elsa.Workflows.Management.Providers;
+
+/// <summary>
+/// Provides activity descriptors based on a list of NON-ELSA activity types finded into the same directory of the assembly.
```

Comment:
Based on the discussion, is it correct that this filtering out of Elsa activities is t prevent them from being added twice, which would lead into an exception? If yes, then let's update the service that invokes these providers by simply ignoring already-registered activities.